swap_meet/decor.py

```diff
@@ -0,0 +1,22 @@
+class Decor:
```

Comment:
This is the first of many changes that'd need to be made to this class definition to properly inherit from the Item class.
```suggestion
class Decor(Item):
```